app/controllers/tree_display_controller.rb

```diff
@@ -206,8 +206,12 @@ def children_node_ng
     flash[:error] = "Invalid JSON in the TreeList" unless json_valid? params[:reactParams][:child_nodes]
     child_nodes = child_nodes_from_params(params[:reactParams][:child_nodes])
     tmp_res = {}
-    child_nodes.each do |node|
-      initialize_fnode_update_children(params, node, tmp_res)
+    begin
```

Comment:
This change was suggested by the TA Suraj to deal with the child node error in Travis CI.